go.mod

```diff
@@ -8,6 +8,7 @@ replace (
 	github.com/vmware-tanzu/tanzu-framework/apis/run => ./apis/run
 	github.com/vmware-tanzu/tanzu-framework/cli/core => ./cli/core
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ./cli/runtime
+	github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin-admin/builder => ./cmd/cli/plugin-admin/builder
```

Comment:
Is this really needed? I believe no module will be dependent on `builder` plugin, so this might not be needed.

Reply:
Good catch!  This was left-over from my trying to get things to work early on.
Will remove.